### PR TITLE
Create j-michelle-hu.yaml

### PR DESCRIPTION
**Pull request by jmichellehu**

### CODE REVIEW

**Comment by JessicaS11** [comment]
@jmichellehu I'm not sure what's going on (since it's building successfully), but when I click the link to your bio in the preview nothing happens. Can someone check if this is a general issue or only on my end?

**Comment by jomey** [comment]
> @jmichellehu I'm not sure what's going on (since it's building successfully), but when I click the link to your bio in the preview nothing happens. Can someone check if this is a general issue or only on my end?

Only a quick guess, but the `.` in the first name might throw a wrench for opening the modal.

**Comment by jomey** [comment]
> > @jmichellehu I'm not sure what's going on (since it's building successfully), but when I click the link to your bio in the preview nothing happens. Can someone check if this is a general issue or only on my end?
> 
> Only a quick guess, but the `.` in the first name might throw a wrench for opening the modal.

Confirmed. Have to add support in the website logic and improve the handling of names with a `.`

**Review by jomey — APPROVED** [review]

File: team/j-michelle-hu.yaml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    
+  social:
+    # Remove list item (- icon & link) if you don't have an account
+    - icon: github
+      link: https://github.com/jmichellehu
+    - icon: linkedin
+      link: https://www.linkedin.com/in/j-michelle-hu
+  user_groups:
+    # From https://escience.washington.edu/using-data-science/hackweeks/planning/hackweek-roles
+    - Hackweek helper
```
Comment (jomey) [inline team/j-michelle-hu.yaml]: For consistency - I think we have all roles capitalized
```
Hackweek Helper
```

**Pull request merged**